scripts/provisioning/roles/motr-build/vars/RedHat.yml

```diff
@@ -50,6 +50,7 @@ motr_build_deps_pkgs:
   - rpm-build
   - systemd-devel
   - libedit-devel
+  - 'https://www.nasm.us/pub/nasm/releasebuilds/2.14/linux/nasm-2.14-0.fc27.x86_64.rpm'
```

Comment:
@andriytk , @chumakd 
From the base-os repo of CentOS 7, the `nasm-2.10` can be installed.

But that version is too old. To build Intel isa-l, newer version is needed. That's why I am using the newer version from its official site.
But how to ensure the nasm-2.10 from CentOS 7 release is not installed?
If already installed, we may need to remove it first.